python_modules/dagster/dagster_tests/asset_defs_tests/test_asset_graph.py

```diff
@@ -89,7 +89,7 @@ def asset3(asset1, asset2): ...
     assert asset_graph.is_partitioned(asset1.key)
     assert asset_graph.have_same_partitioning(asset1.key, asset2.key)
     assert not asset_graph.have_same_partitioning(asset1.key, asset3.key)
-    assert asset_graph.get_children(asset0.key) == {asset1.key, asset2.key}
+    assert asset_graph.get_child_asset_keys(asset0.key) == {asset1.key, asset2.key}
```

Comment:
`get_children` callsites needed to be changed because `get_children` now returns nodes instead of keys